cosmos/elystestnet.json

```diff
@@ -1,13 +1,15 @@
 {
   "chainId": "elystestnet-1",
-  "chainName": "elys",
+  "chainName": "Elys Network Testnet",
   "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/elystestnet/chain.png",
-  "rpc": "https://elys-testnet-rpc.staketab.org:443",
-  "rest": "https://elys-testnet-rest.staketab.org",
+  "rpc": "https://rpc.testnet.elys.network:443",
+  "rest": "https://api.testnet.elys.network",
+  "walletUrl": "https://testnet.elys.network",
```

Comment:
walletUrl was deprecated, so could you please remove this line?

Reply:
Done thank you